README.md

```diff
@@ -8,174 +8,23 @@ programming as well as a community of excited developers.
 Think this is cool and want to get involved? Email us at auburnacm@gmail.com
 or join us on Slack! We'd love you to help us out!
 
-## Configuring your project
+Just a quick note: we only support Ubuntu and MacOS for development. If you use any other development platform, we will not be able to assist you.
 
-Since we don't want to share config files (that's bad), you should create your
-own `config.py` file when once you clone the repository. There are a few things
-you should drop in there. A good config file looks like this:
+## Mac Installation
 
-```py
-# This file holds constants for configuration.
-# DO NOT EXPOSE THIS TO GITHUB.
-DEBUG = True
-SECRET_KEY = 'developement key'
-DATA_FOLDER = '/Path/to/your/clone/auacm/auacm/app/data'
-TEMP_DIRECTORY = '/Path/to/your/clone/auacm/auacm/temp'
+1. cd to the `/../auacm/setup/` directory
 
-TEST_USERNAME = 'will'
-TEST_PASSWORD = 'password'
-```
-Your `config.py` file belongs in the `auacm/auacm/app` directory.
+2. execute the script `./macos_setup.py`. This will walk you through setting up the development environment for the site.
```

Comment:
Is this a shell script or a python file? This and the installation instructions disagree.